src/main/java/touro/alter/GraphView.java

```diff
@@ -0,0 +1,29 @@
+package touro.alter;
+
+import org.graalvm.compiler.graph.Graph;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class GraphView extends JComponent {
+
+    private final Color NODE_COLOR = Color.DARK_GRAY;
+    private final Color SINGLE_POINT_OF_FAILURE = Color.RED;
+    private final Color NODE_OUTLINE_COLOR = Color.BLACK;
+    private final Color NODE_TEXT_COLOR = Color.WHITE;
+    private final Graph graph;
```

Comment:
Do we need another color for lines between nodes? To show a connection between them?

Reply:
Definitely could be. I'll take it into account as I continue to work on view